<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SUN-HSINGH2" ignored="0" name="Test" tests="2" failures="2" timestamp="2022-05-26T20:50:44 EDT" time="1.846" errors="0">
  <testcase name="register" time="0.066" classname="ExtentReport.demoTesCase">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ExtentReport.demoTesCase.register(demoTesCase.java:48)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- register -->
  <testcase name="login" time="0.004" classname="ExtentReport.demoTesCase2">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ExtentReport.demoTesCase2.login(demoTesCase2.java:40)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
